internal/serve/httphandler/receiver_send_otp_handler.go

```diff
@@ -70,6 +70,13 @@ func (h ReceiverSendOTPHandler) ServeHTTP(w http.ResponseWriter, r *http.Request
 		return
 	}
 
+	if phoneValidateErr := utils.ValidatePhoneNumber(receiverSendOTPRequest.PhoneNumber); phoneValidateErr != nil {
+		phoneValidateErr = fmt.Errorf("validating phone number %s: %w", utils.TruncateString(receiverSendOTPRequest.PhoneNumber, len(receiverSendOTPRequest.PhoneNumber)/4), phoneValidateErr)
+		log.Ctx(ctx).Error(phoneValidateErr)
+		httperror.BadRequest("request invalid", phoneValidateErr, map[string]interface{}{"phone_number": "invalid phone number provided"}).Render(w)
+		return
+	}
+
```

Comment:
TBH, I think you can remove the `receiverSendOTPRequest.PhoneNumber != ""` check because that is already caught in the `utils.ValidatePhoneNumber` method.

Reply:
Sounds good to me!! 🚀 

I did this way because I was leveraging returning a better message (BadRequest). But, we can return a specific error from the `util` function when the phone number is empty instead and still return a bad request response.

Reply:
Changed: [575bd7d](https://github.com/stellar/stellar-disbursement-platform-backend/pull/38/commits/575bd7d38bfb8965493e6fde3e32de8019852125)